src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx

```diff
@@ -265,11 +424,11 @@ export default class ExpressionEnrichmentContainer extends React.Component<IExpr
                     </div>
                     <ExpressionEnrichmentTable
                         data={this.filteredData}
-                        onCheckGene={this.props.store ? this.onCheckGene : undefined}
-                        onGeneNameClick={this.props.store ? this.onGeneNameClick : undefined}
+                        onCheckGene={this.props.queriedHugoGeneSymbols !== undefined ? this.onCheckGene : undefined}
+                        onGeneNameClick={this.onGeneNameClick}
                         dataStore={this.dataStore}
                         mutexTendency={this.props.alteredVsUnalteredMode}
-                        checkedGenes={this.props.store ? this.checkedGenes : undefined}
+                        checkedGenes={this.props.queriedHugoGeneSymbols !== undefined ? this.checkedGenes : undefined}
```

Comment:
seems weird to me to use this.checkedGenes if queriedHugoGeneSymbols is undefined, are they related?

Reply:
they are related. gene checkboxes are enabled only for results page. and `queriedHugoGeneSymbols` is defined only for results page. Mya be I need to be more specific and create another property just for this